api_validation/README.md

```diff
@@ -15,7 +17,11 @@ It requires cudf, rapids-4-spark and spark jars.
 
 ```
 cd api_validation
-mvn scala:run
+// To run validation script on all version of Spark-3(3.0.0, 3.0.1-SNAPSHOT and 3.1.0-SNAPSHOT)
```

Comment:
did you mean to have the -3 on here?

Reply:
I wanted to mean it as different versions of Spark 3. Removed it as it might be confusing.